HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -144,7 +144,6 @@ def customiseFor2018Input(process):
 
     return process
 
-
```

Comment:
Please now restore the original file here so that there is no spurious change.

Reply:
Done